relations/services/create.py

```diff
@@ -4,26 +4,79 @@
 from typing import Optional
 from relations.domain import ArXivID, EPrint, Relation, RelationID, \
     RelationType, Resource
-from .model import db, RelationDB
+from .model import db, RelationDB, ActivationDB
+from .util import relation_from_DB
 
 
 class StorageError(Exception):
     """Error that happens in strage of data."""
 
-    pass
-
 
 def create(arxiv_id: ArXivID,
            arxiv_ver: int,
-           relation_id: Optional[RelationID],
-           relation_type: RelationType,
            resource_type: str,
            resource_id: str,
            description: str,
            creator: Optional[str]) -> Relation:
     """
     Create a new relation for an e-print.
 
+    Parameters
+    ----------
+    arxiv_id: ArXivID
+        The arXiv ID of the e-print.
+    arxiv_ver: int
+        The version of the e-print.
+    resource_type: str
+        The type of the corresponding resource.
+    resource_id: str
+        An identifier of the resource e.g., DOI.
+    description: str
+        A description for the relation.
+    creator: Optional[str]
+        Info of the user/app who requested this relation creation.
+
+    Returns
+    -------
+    Relation
+        The newly-created relation.
+
+    """
+    # store it to DB
+    rel_data = RelationDB(rel_type=RelationType.ADD,
+                          arxiv_id=str(arxiv_id),
+                          arxiv_ver=arxiv_ver,
+                          resource_type=resource_type,
+                          resource_id=resource_id,
+                          description=description,
+                          added_at=datetime.now(),
```

Comment:
It occurs to me that we will want to localize these ``datetime``s to UTC. In other apps we've been using ``pytz`` for this.

```python
from pytz import UTC

...

    added_at=datetime.now(UTC)
```